### ISSUE TITLE
bug report

### ISSUE THREAD

**Issue by Rubenllg** [fds2610/NCT-Desktop#28]
Greetings, 
we are using this application atm. and we discoverd some little issues.
1. the date/ time in the notifier won't set to the current locations time
2. the notifications for video calls dont work with the windows notifications

i really look foreward to use the messenger in the future

**Comment by fds2610** [comment]
Hi Ruben,
in Windows current client version 3.0.1 shows server notifications - so there might be no more use for my App...
still need the fixes ?

**Comment by Rubenllg** [comment]
The goal was to use ur app as an alternative to ms teams etc. So it is not likely to happen anymore? Gesendet mit der GMX Mail AppAm 20.09.20 um 10:32 schrieb Schappi
                
            
            
                
                    Von: "Schappi" <notifications@github.com>Datum: 20. September 2020An: "fds2610/NCT-Desktop" <NCT-Desktop@noreply.github.com>Cc: "Author" <author@noreply.github.com>,"Rubenllg" <ruben.bartling@gmx.de>Betreff: Re: [fds2610/NCT-Desktop] bug report (#28)
                
                
Hi Ruben,
in Windows current client version 3.0.1 shows server notifications - so there might be no more use for my App...
still need the fixes ?

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/fds2610/NCT-Desktop/issues/28#issuecomment-695760653",
"url": "https://github.com/fds2610/NCT-Desktop/issues/28#issuecomment-695760653",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by fds2610** [comment]
i will have a look at it later but it will take some time

**Comment by Baccanno** [comment]
> Hi Ruben,
> in Windows current client version 3.0.1 shows server notifications - so there might be no more use for my App...
> still need the fixes ?

Are you sure about this ? I see a talk link appeared in the desktop client but I don't seem to receive notifications

**Comment by fds2610** [comment]
Yes, for sure. with latest NC client I do receive notifications! NC Client features list say: "Receive notifications of server events like a new share or audio/video call" via: https://nextcloud.com/clients/

**Comment by builder-main** [comment]
The very latest NC Client works indeed better. I do receive notifications
